test/test_view/test_project_answer_field_config.py

```diff
@@ -17,8 +17,8 @@ def test_get_config(self):
         project = ProjectFactory.create(published=True)
         url = '/project/%s/answerfieldsconfig?api_key=%s' % (project.short_name, project.owner.api_key)
         res = self.app.get(url)
-        assert 'Answer Field Config' in res.data, res.data
-        assert 'Consensus Config' in res.data, res.data
+        assert '<fields-config' in res.data, res.data
```

Comment:
This looks good. I was just curious if the bracket in the name is required for this assert?

```
<fields-config
<consensus-config
```

Or would the following suffice?

```
fields-config
consensus-config
```

Another idea to consider might be to leave off the bracket and instead check that `res.data` *begins* with the keyword, rather than exists anywhere within the string. Just some ideas. :)

Reply:
Thank you Kory. Yes it may look weird with brackets., the reason I added is that I wanted to ensure <fields-config> and <consensus-config> component exist, otherwise it matches variable names that include the string.
And res.data begins with some header templates that we do not really care in this test case.